src/zcbenchmarks.cpp

```diff
@@ -31,6 +31,31 @@
 #include "zcash/IncrementalMerkleTree.hpp"
 
 using namespace libzcash;
+// This method is based on Shutdown from init.cpp
+void shutdown_wallet()
+{
+    LogPrintf("%s: In progress...\n", __func__);
+    static CCriticalSection cs_Shutdown;
+    TRY_LOCK(cs_Shutdown, lockShutdown);
+    if (!lockShutdown)
+        return;
+
+if (pwalletMain)
+    pwalletMain->Flush(false);
```

Comment:
Indentation (both lines)

Reply:
The rest of the method needed indentation